### PR TITLE
Clean Homepage

### PR DESCRIPTION
**Pull request by ananzh**
Before clean, homepage is

Screenshot 2023-02-21 at 15 27 51

After clean, now the homepage is like:

Screenshot 2023-02-21 at 15 18 16

* no elastic image
* no chart

### Issues Resolved
https://github.com/opensearch-project/oui/issues/255
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] All tests pass
  - [ ] `yarn lint`
  - [ ] `yarn test-unit`
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src-docs/src/views/home/home_illustration.tsx [hunk 1]
```diff
@@ -9,20 +9,18 @@
...
-import illustrationDarkMode from '../../images/illustration-oui-hero-500-darkmode-shadow.svg';
-import illustrationLightMode from '../../images/illustration-oui-hero-500-shadow.svg';
-import { OuiImage } from '../../../../src/components/image';
+import React from 'react';
+//import { ThemeContext } from '../../components/with_theme';
+//import { OuiImage } from '../../../../src/components/image';
 
 function Icon() {
-  const themeContext: any = useContext(ThemeContext);
+  //const themeContext: any = useContext(ThemeContext);
```
Comment (kavilla) [inline src-docs/src/views/home/home_illustration.tsx:17]: Honestly thinking we should create a follow up issue to add a logo in the view if we want and delete this un used code.

Or delete this file.

Th

**Comment by kavilla** [comment]
Otherwise instead of changing the file you can actually just comment out (or delete) https://github.com/opensearch-project/oui/blob/main/src-docs/src/views/home/home_view.js#L88 from `OuiFlexItem` to the close and this solves the problem as well

File: src-docs/src/views/home/home_illustration.tsx (hunk 1 above)
Comment (ananzh) [reply src-docs/src/views/home/home_illustration.tsx]: Okay will change

**Comment by ananzh** [comment]
> Otherwise instead of changing the file you can actually just comment out (or delete) https://github.com/opensearch-project/oui/blob/main/src-docs/src/views/home/home_view.js#L88 from `OuiFlexItem` to the close and this solves the problem as well

I kept the [home_illustration.tsx](https://github.com/opensearch-project/oui/pull/500/files/01078e826a2b32905f512631ecd47d2b8d2dc2ea#diff-a6c299de7f5386b1dd112b56308be51ddf3695595fd90e431ffbeb0c5402469b) and remove the whole `OuiFlexItem`.

**Review by kavilla — APPROVED** [review]

File: src-docs/src/views/home/home_illustration.tsx (hunk 1 above)
Comment (KrooshalUX) [reply src-docs/src/views/home/home_illustration.tsx]: I do not want to create a follow up issue to add a graphic here. I am being very judicious with how much time we spend on this forked site in favor of moving over to the storybook solution in the future. Lets move forward with delete.

**Comment by KrooshalUX** [comment]
> > Otherwise instead of changing the file you can actually just comment out (or delete) https://github.com/opensearch-project/oui/blob/main/src-docs/src/views/home/home_view.js#L88 from `OuiFlexItem` to the close and this solves the problem as well
> 
> I kept the [home_illustration.tsx](https://github.com/opensearch-project/oui/pull/500/files/01078e826a2b32905f512631ecd47d2b8d2dc2ea#diff-a6c299de7f5386b1dd112b56308be51ddf3695595fd90e431ffbeb0c5402469b) and remove the whole `OuiFlexItem`.

We can get rid of the home_illustration.tsx, I am not planning to provide a replacement and this is not our original artwork.

**Review by seanneumann — APPROVED** [review]
👍

**Pull request merged**